config/intellij-idea-inspections.xml

```diff
@@ -2814,7 +2815,10 @@
                      enabled_by_default="false"/>
     <inspection_tool class="JavacQuirks" enabled="true" level="ERROR" enabled_by_default="true"/>
     <inspection_tool class="JavadocReference" enabled="true" level="WARNING"
-                     enabled_by_default="true"/>
+                     enabled_by_default="true">
+        <!-- is still provides valuable information to programmer -->
+        <option name="REPORT_INACCESSIBLE" value="false" />
```

Comment:
I am not agree here a bit.

current violations:
```
Declaration has problems in Javadoc references
AbstractJavadocCheck
Symbol 'firstNonTightHtmlTag' is inaccessible from here
UniquePropertiesCheckTest
Symbol 'getLineNumber(FileText, String)' is inaccessible from here
```

> Symbol 'firstNonTightHtmlTag' is inaccessible from here

it is not reasonable to provide link to private field. Second link in this javadoc is valid and only it should be present.
it gives enough context for user.  link to private field is not required.

> Symbol 'getLineNumber(FileText, String)' is inaccessible from here

we need to do suppression of this case, usage of reflection in tests is hack and it is a problem of test, javadoc over test should explain all, and it will contain suppression and explanation of suppression. It is good practice to allow hacks but force engineer to provide a lot of documentation/suppressions to show that it is not easy way :) .

Reply:
Done.